src/main/java/mate/academy/springboot/datajpa/service/CategoryServiceImpl.java

```
@@ -0,0 +1,34 @@
+package mate.academy.springboot.datajpa.service;
+
+import mate.academy.springboot.datajpa.model.Category;
+import mate.academy.springboot.datajpa.repository.CategoryRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CategoryServiceImpl implements CategoryService {
+    private CategoryRepository categoryRepository;
+
+    public CategoryServiceImpl(CategoryRepository categoryRepository) {
+        this.categoryRepository = categoryRepository;
+    }
+
+    @Override
+    public Category save(Category category) {
+        return categoryRepository.save(category);
+    }
+
+    @Override
+    public Category getById(Long id) {
+        return categoryRepository.getById(id);
```

let's throw an exception in case category isn't found